README.md

```diff
@@ -901,6 +901,111 @@ _You can enable the following settings in Xcode by running [this script](resourc
   }
   ```
 
+* <a id='conditional-assignment'></a>(<a href='#conditional-assignment'>link</a>) **When initializing a new property with the result of a conditional statement (e.g. an if or switch statement), use a single if/switch expression where possible** rather than definining an uninitialized property and initializing it on every branch of the following conditional statement. [![SwiftFormat: conditionalAssignment](https://img.shields.io/badge/SwiftFormat-conditionalAssignment-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#conditionalAssignment)
```

Comment:
I wonder if we can have a more specific/descriptive slug than `conditional-assignment` but I don't feel strongly that we need to make a change.